personalwebsite/requirements.txt

```diff
@@ -1,4 +1,9 @@
+asgiref==3.3.1
```

Comment:
What do the 5 new libraries do?

Reply:
I might have accidentally added those, I'll revert this file back to the one in the main branch.